src/tasks/Crossgen2Tasks/ShimFilesSimulatingLogicInSdkRepo/Microsoft.NET.CrossGen.props

```diff
@@ -1,6 +1,6 @@
 <!--
 ***********************************************************************************************
-Microsoft.NET.CrossGen.targets
+Microsoft.NET.CrossGen.props
```

Comment:
I'm fine with this one as it fixes a real typo despite the fact that I find this comment superfluous and I would myself consider deleting it including the asterisk artwork above instead, but if it's supposed to be the style of the code partition, so be it.

Reply:
Files that are part of the .NET SDK usually have this header, I would keep it.